presto-spark-base/src/main/java/com/facebook/presto/spark/execution/BatchTaskUpdateRequest.java

```diff
@@ -26,14 +26,17 @@ public class BatchTaskUpdateRequest
 {
     private final TaskUpdateRequest taskUpdateRequest;
     private final Optional<String> shuffleWriteInfo;
+    private final Optional<String> broadcastBasePath;
 
     @JsonCreator
     public BatchTaskUpdateRequest(
             @JsonProperty("taskUpdateRequest") TaskUpdateRequest taskUpdateRequest,
-            @JsonProperty("shuffleWriteInfo") Optional<String> shuffleWriteInfo)
+            @JsonProperty("shuffleWriteInfo") Optional<String> shuffleWriteInfo,
+            @JsonProperty("broadcastBasePath") Optional<String> broadcastBasePath)
```

Comment:
Can we re-use shuffleWriteInfo? In case of broadcast, it can store the broadcast base path, no?

Reply:
I have kept it separate to avoid mixing up logic. Also, there are existing code paths(logic) which depend on this variable being populated : https://github.com/search?q=repo%3Aprestodb%2Fpresto%20shuffleWriteInfo&type=code